snuba/datasets/transactions_processor.py

```diff
@@ -155,6 +155,24 @@ def process_message(
                     extra={"measurements": measurements},
                     exc_info=True,
                 )
+
+        breakdowns = data.get("breakdowns")
```

Comment:
Sorry, I should have been more clear: Please leave the try...except around the breakdowns for now, so we can track any errors as development happens. Similarly to measurements, we can remove the try...except once we launch to production.

Reply:
Right now if there is a bug with breakdowns, it will prevent the entire event from being processed.

Reply:
Ok. I'll leave the try-catch wrapper in.